### PR TITLE
Updated the caption to be more descriptive

### PR DESCRIPTION
**Pull request by lyman-phillips**

### CODE REVIEW

File: _slides/09-lyman-phillips.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Ninja cats are cool.
```
Comment (hj8642) [inline _slides/09-lyman-phillips.md:6]: nice caption, Lyman

Comment (hj8642) [inline _slides/09-lyman-phillips.md:6]: great caption!

**Review by hj8642 — APPROVED** [review]
looks great, Lyman! Let's go...

**Review by lyman-phillips — COMMENTED** [review]
Looks good. Thanks.

**Comment by lyman-phillips** [comment]
This is approved. Nioce caption,.

**Pull request merged**